src/mpack/lmpack.c

```diff
@@ -246,7 +246,7 @@ static mpack_uint32_t lmpack_objlen(lua_State *L, int *is_array)
   }
 
 end:
-  if ((size_t)-1 > (mpack_uint32_t)-1 && len > (mpack_uint32_t)-1)
+  if ((size_t)-1 > (mpack_uint32_t)-1 && len > (mpack_uint32_t)-1) // -V560
```

Comment:
I don't think `(size_t) - 1 > (mpack_uint32_t) - 1` is always true because the type `mpack_uint32_t` can change depending on platform. I think this is false alarm.

Reply:
Unless PVS *also* takes platform dependence into account, which I don't think so.